AppSigner/MainView.swift

```diff
@@ -50,22 +50,20 @@ class MainView: NSView, URLSessionDataDelegate, URLSessionDelegate, URLSessionDo
 //    let plistbuddyPath = "/usr/libexec/plistbuddy"
     
     //MARK: Drag / Drop
-    @objc var fileTypes: [String] = ["ipa","deb","app","xcarchive","mobileprovision","appex"]
-    @objc var urlFileTypes: [String] = ["ipa","deb"]
+    static let urlFileTypes = ["ipa", "deb"]
+    static let allowedFileTypes = urlFileTypes + ["app", "appex", "xcarchive"]
+    static let fileTypes = allowedFileTypes + ["mobileprovision"]
     @objc var fileTypeIsOk = false
     
     @objc func fileDropped(_ filename: String){
-        switch(filename.pathExtension.lowercased()){
-        case "ipa", "deb", "app", "xcarchive", "appex":
+        switch filename.pathExtension.lowercased() {
+        case let ext where MainView.allowedFileTypes.contains(ext):
```

Comment:
why aren't urlFileTypes also included here?

Reply:
`allowedFileTypes` is a superset of `urlFileTypes`, check line 54